src/main/resources/db/migration/common/V1_110__insert_automated_task_data.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO darts.automated_task (aut_id,task_name,task_description,cron_expression,cron_editable)
+VALUES (1,'ProcessDailyList','Process the latest daily list for each courthouse','0 2 * * * *',true);
```

Comment:
we might not want this running over the weekend.

Reply:
Do you know if any courts are open on the weekend? Is the process of generating the daily list going to only work out the daily list for the open courts? Is there the possibility a case will be heard on a weekend so it would make sense to run even though most the time there won't be any hearings?